Makefile

```diff
@@ -93,6 +93,7 @@ attribution-files:
 	build/update-attribution-files/make_attribution.sh projects/aws/bottlerocket-bootstrap
 	build/update-attribution-files/make_attribution.sh projects/replicatedhq/troubleshoot
 	build/update-attribution-files/make_attribution.sh projects/tinkerbell/cluster-api-provider-tinkerbell
+	build/update-attribution-files/make_attribution.sh projects/tinkerbell/hegel
```

Comment:
A similar entry needs to be added to the `clean` target too.

Reply:
Actually the repo had no `clean` target in its make. Hence i didnt add that.

Reply:
this calls the clean make target from the common makefile, not from upstream. just removes the cloned repo and output folder

Reply:
Git it. Added it in 👍